README.md

```diff
@@ -182,4 +169,14 @@ Sets the frequency at which data is flushed to the server. Resets the interval t
 events.teardown()
 ```
 
-Stops the interval timer, clears the data queue and prevents any further data from being flushed to the server.
+Stops the interval timer, clears the data queue and prevents any further data from being flushed to the server. Useful for testing
+
+#### `events.from`
+
+```js
+const scoped = events.from(ctx);
+```
+
+Returns a scoped version of the events api.
```

Comment:
Explain why one would want to use this over the global instance. Document API differences between a global emitter and a scoped one.

Reply:
That should be explained in the "Accessing `ctx` section